File: Python/Warmup-2/string_splosion.py
"""
Given a non-empty string like "Code" return a string like "CCoCodCode".

string_splosion('Code') → 'CCoCodCode'
string_splosion('abc') → 'aababc'
string_splosion('ab') → 'aab'

"""
def string_splosion(str):
  str1=""
  for i in range(len(str)):
    str1=str1+str[:i]
  return str1+str

File: Java/Warmup-2/stringSplosion.java
/*
Given a non-empty string like "Code" return a string like "CCoCodCode".

stringSplosion("Code") ? "CCoCodCode"
stringSplosion("abc") ? "aababc"
stringSplosion("ab") ? "aab"
*/

public String stringSplosion(String str) {
  String str1="";
  for(int i=0;i<str.length();i++){
    str1=str1+str.substring(0, i);
  }
  return str1+str;
}
